### ISSUE TITLE
The Port Itself: Gamepad/touchscreen shortcut for Android keyboard?

### ISSUE THREAD

**Issue by Shockwave508** [daniele-rapagnani/alephone-android#39]
Is it possible to add a touch/gamepad shortcut to bring up the Android keyboard?

**Comment by daniele-rapagnani** [comment]
Can you clarify a bit what you mean with this?

**Comment by Raypuia** [comment]
He means adding a button that can show up the android default keyboard in both ingame or the menu

**Comment by Shockwave508** [comment]
Exactly. This sort of thing would be a necessity for text chat in multiplayer, as well as utilizing the Yellow Crystal in the Pathways Into Darkness scenario, to communicate with the deceased.
